src/util.ts

```diff
@@ -23,13 +23,15 @@ export const getComponentNameOutline = async (documentUri: vscode.Uri) => {
     return componentNameOutline
 }
 
-export const getComputedOutline = async (documentUri: vscode.Uri) => {
-    const outline = await getNormalizedVueOutline(documentUri)
+const findMe = outline => {
+    const currentOutline = outline?.find(({ name }) => name === 'script')?.children ?? outline
+    const defaultOutline = currentOutline?.find(({ name }) => name === 'default').children
 
-    const computedOutline = outline
-        ?.find(({ name }) => name === 'script')
-        ?.children.find(({ name }) => name === 'default')
-        ?.children.find(({ name }) => name === 'computed')
+    return defaultOutline?.find(({ name }) => name === 'computed')
```

Comment:
@yodakit either hardcode this method in `getComputedOutline` or add second parameter, e.g. `componentOption` which will accept 'methods', 'data', 'name' etc

Reply:
@maIIady Moved to `getComputedOutline` :)